backend/Dockerfiles/Dockerfile.python3

```diff
@@ -1,4 +1,4 @@
-FROM python:3-alpine
+FROM python:3.11-alpine
```

Comment:
Is there some mechanism we have to come back to this?

If we don't have something outside the repo, maybe we can comment the reason that's in the description. Someone in the future might not look at this PR and it won't be immediately apparent why it's pinned to `3.11`

Reply:
Fair, I'll add a comment